actions/go-mod-tidy-vendor/plugin.yaml

```diff
@@ -0,0 +1,10 @@
+version: 0.1
+actions:
+  definitions:
+    - id: go-mod-tidy-vendor
+      display_name: Go Mod Tidy & Vendor
+      description: Runs go mod tidy followed by go mod vendor.
+      run: ./go-mod-tidy-vendor.sh
+      import_to_global: true
```

Comment:
I don't think this field exists on action definitions, but we also don't have good CI checks on PRs into this repo yet.